Entity Explorer - Account.ipynb

```diff
@@ -6,20 +6,23 @@
    "source": [
```

Comment:
<p>Not sure where the Path import is used.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>